platform/core.windows/src/org/netbeans/core/windows/options/LafPanel.java

```diff
@@ -202,8 +203,19 @@ boolean valid() {
 
     private void initLookAndFeel() {
         lafs.clear();
-        for( LookAndFeelInfo i : UIManager.getInstalledLookAndFeels() ) {
-            lafs.add( i );
+        String lafFilter = NbBundle.getMessage(LafPanel.class, "Laf_Filter").trim();
+        if (lafFilter.isEmpty()) {
+            lafs.addAll(Arrays.asList(UIManager.getInstalledLookAndFeels()));
+        } else {
+            LookAndFeelInfo cur = getCurrentLaF();
+            LookAndFeelInfo pref = getPreferredLaF();
+            for (LookAndFeelInfo li : UIManager.getInstalledLookAndFeels()) {
+                String name = li.getClassName();
+                name = name.substring(name.lastIndexOf('.') + 1, name.length());
```

Comment:
Might be simpler to just list the full class name, so we don't have to do any parsing here.

Reply:
It just makes the filter string rather long, and some of these LaFs have moved package in the past - not that that's likely in future.